src/imitation/util/util.py

```diff
@@ -160,6 +160,25 @@ def endless_iter(iterable: Iterable[T]) -> Iterator[T]:
     return itertools.chain.from_iterable(itertools.repeat(iterable))
 
 
+def safe_to_tensor(numpy_array: np.ndarray) -> th.Tensor:
```

Comment:
Consider adding `device` flag? From https://pytorch.org/docs/stable/generated/torch.as_tensor.html it seems that when `th.as_tensor` copies, it does so like `data.to(dtype=dtype, device=device)`. I'd expect `data.to(dtype=dtype).to(device=device)` to be slower in many cases (copying twice).

Reply:
Done